Update GitHubPRComment task documentation to include logging feature (#40)
southworkscom/SOUTHWORKS-azure-pipelines-tasks: Tasks/GitHubPRComment/readme.md

Discarding previous automatic comments.
![PipelineResults2](https://user-images.githubusercontent.com/64815358/182950960-3a27e33e-74c6-426a-ba63-f59777b2e310.jpg)

> ## Debug
> 
> The task can be debugged by activating the "Enable System Diagnostics" option in the pipeline before running it.

Reviewer: I'd change this:
```suggestion
## Debug Mode

The task can be debugged by activating the "Enable System Diagnostics" option in the pipeline.
```

Author: Done! Thanks.
